docs/dApps/unity/quickstart.md

```diff
@@ -0,0 +1,373 @@
+
+1. Verify that the SDK works in your project by running the WalletConnection example scene, which demonstrates how to connect to a Tezos wallet in a Unity project:
+
+   1. In the Project panel, expand **Assets > TezosSDK > Examples > WalletConnection** and open the `_WalletConnection` scene.
+   1. Run the scene, which shows a QR code.
+   1. Use your wallet app to scan the barcode and confirm the connection to the Unity project.
+   The project UI shows the address of the connected account, as in this picture:
+
+      <img src="/img/dApps/unity-wallet-connection-scene-connected.png" alt="The new RPC node selected in the Temple wallet" style={{width: 500}} />
+
+## Connecting to wallets
+
+Connecting to a user's wallet is a prerequisite to working with Tezos in any application.
+Accessing the wallet allows your project to see the tokens in it and to prompt the user to submit transactions, but it does not give your project direct control over the wallet.
+Users must still confirm all transactions in their wallet application.
+
+Using a wallet application in this way saves you from having to implement payment processing and security in your application.
+Game developers can also use the wallet and its account as a unique account identifier and as the user's inventory.
+
+1. Copy the `MainThreadExecutor`, `TezosAuthenticator`, and `TezosManager` prefabs to your scene.
+These prefabs provide prerequisites to use Tezos in a scene.
+
+   The `TezosAuthenticator` prefab automatically adds features that connect to users' wallets.
+   If you copy these prefabs into your scene and run it, it shows a QR code that Tezos wallet applications can scan to connect with the application.
+   You can access these features through the prefab and change how the project manages its connection to users' wallets.
+
+   For example, the `QRCode` object receives a handshake from the `TezosManager` object and uses the handshake information to generate a URI and encode that URI to a QR code image:
+
+   ```csharp
+   public void SetQrCode(HandshakeData handshakeData)
+   {
+       var uri = "tezos://?type=tzip10&data=" + handshakeData.PairingData;
+       EncodeTextToQrCode(uri);
+   }
+   ```
+
+   You can adjust this code and the bound Unity object to control when and where the QR code appears.
+
+   The `TezosManager` fields control what users see in their wallet applications before connecting to the project, as shown in this picture of the Inspector panel:
+
+   ![The Inspector panel, showing information about the project](/img/dApps/unity-inspector-tezosmanager.png)
+
+1. Add features to your project to use the connected account.
+For example, the `TezosSDK/Examples/Common/Scripts/AccountInfoUI.cs` file responds to the `AccountConnected` event, which runs when the user scans the QR code and approves the connection in their wallet app.
+You can use this event to get the address of the connected account, as in these steps:
```

Comment:
We can skip these steps and just show the code below. We assume good Unity knowledge for readers, and we want to keep the Quickstart as concise as possible.

Reply:
Trimmed this section.